crates/common/src/ckb_decimal.rs

```diff
@@ -0,0 +1,34 @@
+use gw_types::U256;
+
+// NOTE: u64::MAX is about 10^18, U256::MAX is about 10^77. Multiple 10 pow
+// `CKB_DECIMAL_POW_EXP` should not overflow.
+pub const CKB_DECIMAL_POW_EXP: u32 = 10;
+pub const CKB_DECIMAL_POWER_TEN: u64 = 10u64.pow(CKB_DECIMAL_POW_EXP);
+
+pub fn to_18(amount: u64) -> U256 {
```

Comment:
```suggestion
pub fn to_u256(amount: u64) -> U256 {
```

Reply:
I prefer keep decimal info in function name. Since every sudt amount is U256 now. ```ckb_decimal::to_18``` is better than ```ckb_decimal::to_u256```.